src/views/TransferStatus.tsx

```diff
@@ -49,14 +49,23 @@ export const TransferStatusToast = ({
 
   if (!status) return <LoadingDots size={3} />;
 
+  const inProgressStatusString =
+    type === 'deposit'
+      ? secondsLeft > 0
+        ? STRING_KEYS.DEPOSIT_STATUS
+        : STRING_KEYS.DEPOSIT_STATUS_SHORTLY
+      : secondsLeft > 0
+      ? STRING_KEYS.WITHDRAW_COMPLETE
```

Comment:
is this supposed to be `WITHDRAW_COMPLETE`? or `WITHDRAW_STATUS`?

Reply:
Nice catch! I was waiting for v4-localization, and haven't tested the strings 😅